packages/admin-ui/src/IconPicker/primitives/IconPickerPrimitive.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useRef } from "react";
+import { inputVariants } from "~/Input";
+import { Popover } from "~/Popover";
+import { cn, cva, type VariantProps } from "~/utils";
+import { IconPickerGrid, IconPickerInput, IconPickerTrigger } from "./components";
+import { IconDto } from "../domains";
+import { useIconPicker } from "./useIconPicker";
+
+const iconPickerVariants = cva("wby-cursor-pointer wby-text-neutral-strong", {
+    variants: {
+        size: {
+            md: "wby-w-[64px]",
```

Comment:
I guess we also use regular number here instead of px? Just for consistency maybe? 🤔